google/cloud/vmwareengine/CMakeLists.txt

```diff
@@ -0,0 +1,205 @@
+# ~~~
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ~~~
+
+include(GoogleapisConfig)
+set(DOXYGEN_PROJECT_NAME "VMware Engine API C++ Client")
+set(DOXYGEN_PROJECT_BRIEF "A C++ Client Library for the VMware Engine API")
+set(DOXYGEN_PROJECT_NUMBER "${PROJECT_VERSION}")
+set(DOXYGEN_EXCLUDE_SYMBOLS "internal")
+set(DOXYGEN_EXAMPLE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/quickstart)
+
+unset(mocks_globs)
```

Comment:
Ditto, in most cases unrolling these loops will be easier to grok.

Reply:
Ack.